### PR TITLE
Add warning GTDB-TK results will be empty if no contigs pass completeness filter

### PR DESCRIPTION
**Pull request by jfy133**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/mag/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/mag _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Added`
 
+- [#548](https://github.com/nf-core/mag/pull/548) fixes to GTDBK-TK execusion, CAT/QUAST/DEPTH bin summary file name collisions, BUSCO database parsing, correct CAT name files (reported by @maxibor, @PPpissar, @muniheart, @llborcard, fix by @maxibor)
+- [#564](https://github.com/nf-core/mag/pull/558) - Add warning of empty GTDB-TK results if no contigs pass completeness filter (fix by @jfy133 and @maxibor)
```
Comment (CarsonJM) [inline CHANGELOG.md:11]: Should this be #565 and (https://github.com/nf-core/mag/pull/565)?

**Review by CarsonJM — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jfy133) [inline CHANGELOG.md:11]:
```suggestion
- [#565](https://github.com/nf-core/mag/pull/565) - Add warning of empty GTDB-TK results if no contigs pass completeness filter (fix by @jfy133 and @maxibor)
```

Comment (jfy133) [reply CHANGELOG.md:11]: One day I'll guess right and update both numbers :sweat_smile:

**Comment by jfy133** [comment]
@nf-core-bot fix linting

**Pull request merged**